setup.py

```diff
@@ -45,15 +45,7 @@ def find_meta(meta):
 
 
 URI = find_meta("uri")
-LONG = (
-    read_file("README.rst") + "\n\n" +
-    "Release Information\n" +
-    "===================\n\n" +
-    re.search(r"(\d{2}.\d.\d \(.*?\)\n.*?)\n\n\n----\n",
-              read_file("CHANGELOG.rst"), re.S).group(1) +
-    "\n\n`Full changelog " +
-    "<{uri}en/stable/changelog.html>`_.\n\n"
-).format(uri=URI)
+LONG = 'nothing relevant'
```

Comment:
Why was this change made?

Reply:
Not intended for the PR (I'll remove it as soon as possible).

The error occurs while installing PyOpenSSL in Windows machines (I tried in 3 different machines). The command that fails is:
```bash
python setup.py --install
```
With the output:
```
Traceback (most recent call last):
File "<string>", line 1, in <module>
File "C:\Users\USER\pyopenssl\setup.py", line 54, in <module>
"\n\n`Full changelog " +
AttributeError: 'NoneType' object has no attribute 'group'
```